src/js/containers/search/modals/fullDownload/DownloadBottomBarContainer.jsx

```diff
@@ -108,6 +109,7 @@ export class DownloadBottomBarContainer extends React.Component {
         this.request.promise
             .then((res) => {
                 this.props.setDownloadExpectedFile(res.data.file_name);
+                this.props.setDownloadExpectedUrl(res.data.url);
                 this.checkStatus();
```

Comment:
In the `downloadFile()` method of this component, can you ensure that it closes the download modal if it is open when the download occurs?

Currently, the Redux store is reset, so the modal stays open but clears itself.

![screen shot 2018-01-02 at 11 56 35 am](https://user-images.githubusercontent.com/3594363/34492478-bd9dc92e-efb5-11e7-88f3-38595e7f69f4.png)

Reply:
auto closes when download completes